pkg/kv/kvserver/logstore/sideload.go

```diff
@@ -23,7 +23,7 @@ import (
 	"go.etcd.io/etcd/raft/v3/raftpb"
 )
 
-var errSideloadedFileNotFound = errors.New("sideloaded file not found")
+var ErrSideloadedFileNotFound = errors.New("sideloaded file not found")
```

Comment:
Doesn't this need a comment or linters will complain?

Reply:
Huh, apparently not.

Reply:
Yeah, maybe this lint is disabled. I'm unexporting this though, it's used only inside the package. Will now revisit/fix the exporting/commenting in this PR, and then merge.